### ISSUE TITLE
Wagtail docker demo cannot load initial data

### ISSUE THREAD

**Issue by YuriyOrlov** [wagtail/bakerydemo#229]
I've built container, but when I've tried to load initial data this is what happend
```
Starting wagtail_demo_db_1            ... done
Starting wagtail_demo_redis_1         ... done
Starting wagtail_demo_elasticsearch_1 ... done
                                 List of databases
   Name    |  Owner   | Encoding |  Collate   |   Ctype    |   Access privileges   
-----------+----------+----------+------------+------------+-----------------------
 app_db    | app_user | UTF8     | en_US.utf8 | en_US.utf8 | 
 postgres  | app_user | UTF8     | en_US.utf8 | en_US.utf8 | 
 template0 | app_user | UTF8     | en_US.utf8 | en_US.utf8 | =c/app_user          +
           |          |          |            |            | app_user=CTc/app_user
 template1 | app_user | UTF8     | en_US.utf8 | en_US.utf8 | =c/app_user          +
           |          |          |            |            | app_user=CTc/app_user
(4 rows)

Postgres is up - continuing
/venv/lib/python3.5/site-packages/dotenv.py:56: UserWarning: Not reading .env - it doesn't exist.
  warnings.warn("Not reading {0} - it doesn't exist.".format(dotenv))
SystemCheckError: System check identified some issues:

ERRORS:
wagtailimages.Image.file: (fields.E210) Cannot use ImageField because Pillow is not installed.
        HINT: Get Pillow at https://pypi.org/project/Pillow/ or run command "pip install Pillow".
wagtailimages.Rendition.file: (fields.E210) Cannot use ImageField because Pillow is not installed.
        HINT: Get Pillow at https://pypi.org/project/Pillow/ or run command "pip install Pillow".
wagtailusers.UserProfile.avatar: (fields.E210) Cannot use ImageField because Pillow is not installed.
        HINT: Get Pillow at https://pypi.org/project/Pillow/ or run command "pip install Pillow".
```
But Pillow lib was collected and installed properly
```
Collecting Pillow==4.0.0 (from -r /requirements/base.txt (line 5))
  Downloading https://files.pythonhosted.org/packages/8d/80/eca7a2d1a3c2dafb960f32f844d570de988e609f5fd17de92e1cf6a01b0a/Pillow-4.0.0.tar.gz (11.1MB)
```
but if you run 
```
docker-compose run app /venv/bin/python -m pip install Pillow
```
it says, that Pillow already installed
```
Starting wagtail_demo_elasticsearch_1 ... done
Starting wagtail_demo_redis_1         ... done
                                 List of databases
   Name    |  Owner   | Encoding |  Collate   |   Ctype    |   Access privileges   
-----------+----------+----------+------------+------------+-----------------------
 app_db    | app_user | UTF8     | en_US.utf8 | en_US.utf8 | 
 postgres  | app_user | UTF8     | en_US.utf8 | en_US.utf8 | 
 template0 | app_user | UTF8     | en_US.utf8 | en_US.utf8 | =c/app_user          +
           |          |          |            |            | app_user=CTc/app_user
 template1 | app_user | UTF8     | en_US.utf8 | en_US.utf8 | =c/app_user          +
           |          |          |            |            | app_user=CTc/app_user
(4 rows)

Postgres is up - continuing
Requirement already satisfied: Pillow in /venv/lib/python3.5/site-packages (4.0.0)
Requirement already satisfied: olefile in /venv/lib/python3.5/site-packages (from Pillow) (0.46)

```

Any suggestions ?

**Comment by fred104** [comment]
I encountered the same issue and have a fix for that. Check this PR. https://github.com/wagtail/bakerydemo/pull/230

**Comment by gasman** [comment]
Fixed in #230
